capstone/src/main/webapp/script.js

```diff
@@ -12,11 +12,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+window.addEventListener('DOMContentLoaded', (event) => {
+  if (document.getElementById('map')) {
+    createMap();
+  }
+  if (document.getElementById('filters-form')) {
+    getInputFilters();
+  }
+});
+
+var map;
```

Comment:
any reason why we are using var over let?

Reply:
changing to let

Reply:
Should we use let for map and center as well? I'd recommend running through the script to make sure everything is either let or const.